### PR TITLE
Add languages to engine json

### PR DESCRIPTION
**Pull request by mrb**
/c @brynary

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -244,6 +245,7 @@ The following fields are declared the specification file, and all are required:
 * `maintainer` (`Object`) - data about the engine maintainer
   * `name` (`String`) - the name of the maintainer
   * `email` (`String`) - the email address of the maintainer
+* `languages` (`[String]`) - an array of programming languages that this engine is meant to analyze. Language agnostic engines can specify `"*"`
```
Comment (brynary) [inline SPEC.md]: What are the acceptable values for `languages`? Is it `Javascript`, `JS` or `JavaScript`?

**Comment by mrb** [comment]
@brynary Provocative question re: possible values for `languages`. I made a pass here, looking at current and imminently supported languages and frameworks. It seems like `supported languages` is something that might belong somewhere else in the spec, and referenced here?

**Comment by brynary** [comment]
@mrb what if we use this list, which GitHub uses to driver their display and source listings:

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

**Comment by mrb** [comment]
@brynary WFM!

**Comment by brynary** [comment]
OK, cool. With that change, this LGTM

**Pull request merged**